requirements.txt

```diff
@@ -61,3 +61,5 @@ disposable-email-domains
 # It is not included by default because of the GPL license conflict.
 # ldap3==2.2.4
 gevent==1.4.0
+importlib-metadata==0.8
```

Comment:
Maybe we should add a comment that this can be removed when upgrading to latest Python?